frontend/src/__tests__/cypress/cypress/tests/mocked/modelCatalog/modelCatalog.cy.ts

```diff
@@ -209,7 +210,7 @@ describe('Model Catalog loading states', () => {
       },
     );
     modelCatalog.visit();
-    cy.contains('Request access to model catalog').should('exist');
+    modelCatalog.findModelCatalogEmptyState();
```

Comment:
@YuliaKrimerman you still need the `.should('exist')` 🙈 I meant to turn the line back to how it was, sorry